scipy/stats/_continuous_distns.py

```diff
@@ -8854,6 +8854,206 @@ def _updated_ctor_param(self):
         return dct
 
 
+class studentized_range_gen(rv_continuous):
+    """
+    The generalized studentized range continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t: student's t distribution
+
+    Notes
+    -----
+    When `v` is arbitrarily large, in this implementation greater than or equal
+    to 2000, an asymptotic method is used in calculation of the cumulative
+    distribution function. [4]_
```

Comment:
When `v` exceeds ___, an asymptotic approximation (infinite degree of freedom) is used to compute the cumulative distribution function.